tox.ini

```diff
@@ -1,19 +1,28 @@
 [tox]
 envlist =
     py{27,35}-codestyle,
-    py{27,35}-django{18,19,110}
+    py{27,35}-django{18,19,110,111}
 
 [testenv]
 setenv =
     PYTHONDONTWRITEBYTECODE=1
-install_command = pip install --no-deps {opts} {packages}
+install_command = pip install {opts} {packages}
```

Comment:
Yeah as noted in the PR description, I'd decided against including in requirements.txt, since it's not a test-only requirement.

However I've updated the PR to make that change now anyway.